### ISSUE TITLE
[Feature request] Color freeze from original Markmap

### ISSUE THREAD

**Issue by fabfabretti** [typora-community-plugin/typora-plugin-markmap#1]
Hi! First of all, thank you for the extremely useful plugin.

I wanted to ask if it was possible or if there were any plans to add some of the other features available on the official [markmap test site](https://markmap.js.org/repl#?d=gist%3Aaf76a4c245b302206b16aec503dbe07b); specifically, I think it'd be very useful to add the colorFreezeLevel, which in original markmap can be specified with
``` markmap
---
title: markmap
markmap:
  colorFreezeLevel: 2
  color:#000000
---
<markmap map here>
```
If it's easier, it would be useful even just being able to specify such options globally, from the extension's settings panel.
Thank you a lot again for your work!

**Comment by plylrnsdy** [comment]
The new version is released!

Now you can update typora-community-plugin core to v2.3.33 and then update markmap in plugin marketplace.

**Comment by fabfabretti** [comment]
Oh that was fast, thank you a lot!

I've updated typora-community-plugin and this plugin as well. Is the syntax for enabling options the same as the markmap REPL? At the moment, I've been trying to use the same one with no results.

Additional info should they be useful:

* Plugin version:
![immagine](https://github.com/user-attachments/assets/0b320715-4348-4315-b32f-7402a5ce92b2)
![immagine](https://github.com/user-attachments/assets/8fe9c2f9-5f5d-4587-8ec5-f80d4bf9e14d)

* Typora+markmap:
![immagine](https://github.com/user-attachments/assets/bbe41313-f39d-42d1-ae80-dd0655a9140d)

* Markmap REPL:
![immagine](https://github.com/user-attachments/assets/efdfa3ac-2a82-498d-a1d8-656ff51ec61d)

**Comment by plylrnsdy** [comment]
No, you just need to use markmap options in the front matter

```markmap
---
colorFreezeLevel: 2
---

# Title

- item 1
- item 2
- item 3
```

In Settings Modal

```
colorFreezeLevel: 2
```

**Comment by fabfabretti** [comment]
Thank you for the clarification! It still doesn't seem to work. Is there a cache or something similar I may need to clear? Or, is there any additional info I can provide?
![immagine](https://github.com/user-attachments/assets/d98fb8ab-9c84-4fdf-9249-06840ce9d751)

**Comment by plylrnsdy** [comment]
It should be updated after editing:

![image](https://github.com/user-attachments/assets/c82760a8-23db-4409-806d-fdbbe61a119a)

You can try to reopen this file

**Comment by fabfabretti** [comment]
I've tried editing the graph, closing and reopening the file and also closing and reopening typora; but I still can't seem to have the color change:
![immagine](https://github.com/user-attachments/assets/3322d860-4618-4403-bd05-3ad7261563c6)
I have also changed theme, just in case it was my custom theme messing with the colors, but it doesn't seem to fix it either:
![immagine](https://github.com/user-attachments/assets/6cadc90c-b652-45d8-9d17-443e0d599b0a)
(Github theme)

Let me know if there's any additional info I can provide, and again thank you a lot!


edit: should this be of use, this is the dev console:
![immagine](https://github.com/user-attachments/assets/5cd78313-57c7-408a-9d6d-f696c5c3e11a)

**Comment by plylrnsdy** [comment]
Check your markmap global options in Settings Modal → MarkMap. You can try to empty it to reset.

**Comment by fabfabretti** [comment]
My previous options were
![immagine](https://github.com/user-attachments/assets/1a206ade-9b80-4f2c-9b47-f6dab116bd7c)
I have emptied them and restarted typora, but the result is still the same:
![immagine](https://github.com/user-attachments/assets/6ef1d4a8-9046-4920-af2c-9c52ea0a197e)

**Comment by plylrnsdy** [comment]
can you try to remove the whitespace before the options

```markmap
---
colorFreezeLevel: 1
---
```

or I make a debug version tomorrow.

**Comment by fabfabretti** [comment]
![immagine](https://github.com/user-attachments/assets/b919b332-3b49-4238-a1fb-da5eb2c1ad4d)
Doesn't seem to fix it for me, sadly.

Take all the time you need, there is no rush! And again, thank you so much for your help.

**Comment by plylrnsdy** [comment]
here is a debug version, it will print some message in dev tools after rendered markmap

[plugin-markmap-v1.1.0-beta.1.zip](https://github.com/user-attachments/files/18272718/plugin-markmap-v1.1.0-beta.1.zip)

**Comment by fabfabretti** [comment]
Hi again, thank you! This is a screenshot of the markmap and the dev console.
![immagine](https://github.com/user-attachments/assets/a377cd97-5407-4c7b-b3e3-278cd9a337e2)

**Comment by plylrnsdy** [comment]
fix in v1.1.1

**Comment by fabfabretti** [comment]
It works now! :D 
Thank you so much for your patience!
